lib/CapabilityDelegation.js

```diff
@@ -38,10 +38,13 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
    *   the creation of the proof.
    * @param [maxTimestampDelta] {integer} a maximum number of seconds that
    *   the date on the signature can deviate from, defaults to `Infinity`.
+   * @param {function} [inspectCapabilityChain] - An async function used to
+   *  check the capability chain. It can be use to find revocations related
+   *  to any of the capabilities in the chain.
```

Comment:
Ideally would probably use some jsdoc features, but at least have quick doc on the function "{capabilityIds}" input and "{valid, error}" output. Not sure how to easily write that but would help to avoid chasing down tests and code to see how to use it.